lib/boku/client.py

```diff
@@ -234,18 +234,30 @@ def start_transaction(self, callback_url, consumer_id,
         :param service_id: The Boku ID (integer) for the service being sold.
         :param country: The ISO 3166-1-alpha-2 country code that they
                         buyer is in.
+        :param product_name: The name of the item being purchased.
         """
         if currency not in constants.DECIMAL_PLACES:
             raise KeyError(
                 'No multiplier to figure out decimal places for currency {c}'
                 .format(c=currency))
         price = int(price * constants.DECIMAL_PLACES[currency])
+
+        if len(product_name) > 20:
+            log.warning('Truncating product name for Boku. Long name: {n}. '
+                        'Transaction: {t}'.format(n=product_name,
+                                                  t=external_id))
+            product_name = product_name[0:20]
+
```

Comment:
What boku won't truncate it in the ui? bah! len is always a pain for some whacky unicode strings.